native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java

```diff
@@ -468,10 +469,12 @@ private static String getAnnotatedColumnName(StructureType streamConstraint, Str
         return null;
     }
 
-    public static void updateBallerinaRecordFields(AbstractResultParameterProcessor resultParameterProcessor,
```

Comment:
does this public function have any usages in other SQL connectors?

Reply:
No. Tested the new sql package against the SQL connectors, all test cases passed.